Enhance the Clamor README

satendrapalsingh opened the pull request:
Add standard README sections like Introduction, Installation, Usage, License, etc.

satendrapalsingh committed:
Enhance the Clamor README

Add standard README sections like Introduction, Installation, Usage, License, etc.

sinkingsugar commented on README.md:
```diff
@@ -1,16 +1,69 @@
+For an example of such an application, see [this](https://github.com/fragcolor-xyz/clamor-frontend). 
+
+## License
+Clamor source code is licensed under the [The Unlicense](./LICENSE).
```
Actually it's complex.
You should mention our pallets are BUSL-1.0.

satendrapalsingh replied:
BUSL-1.0 seems to be [deprecated](https://mariadb.com/bsl-faq-adopting/#:~:text=BSL%20version%201.0%20is%20now%20deprecated) and the official link to the license is only for [BUSL-1.1](https://spdx.org/licenses/BUSL-1.1.html) now. No link for BUSL-1.0.

Should I mention BUSL-1.0 and link to the BUSL-1.1 instead? or just mention BUSL-1.0 without any link?
*I'm assuming we are not immediately updating the pallets license to BUSL-1.1

satendrapalsingh replied:
Just checked, the pallets use BUSL-1.1 license. So will update README accordingly.

satendrapalsingh replied:
Pallets license info added to README.

sinkingsugar commented on README.md:
```diff
@@ -1,16 +1,69 @@
+
+The App Explorer will now connect with your local node and will show the blocks being produced by your node in real-time.
+
+### Connecting to your front-end applicaton
```
wrong, we need to delete clamor-frontend
dotapps is the way.

satendrapalsingh replied:
Polkadot's [dotapps](https://github.com/polkadot-js/dotapps.io)?

satendrapalsingh replied:
Removed the section about connecting own front-end application to Clamor node.

satendrapalsingh committed:
Address review comments

> Remove details about connecting own front-end application to local Clamor instance
> Update the License section with information on the license for Pallets.

sinkingsugar commented on README.md:
```diff
@@ -1,16 +1,63 @@
\ No newline at end of file
+# Clamor
+
+[![license](https://img.shields.io/github/license/fragcolor-xyz/clamor)](./LICENSE)
```
Remove this

satendrapalsingh replied:
done

sinkingsugar commented on README.md:
```diff
@@ -1,16 +1,63 @@
+The App Explorer will now connect with your local node and will show the blocks being produced by your node in real-time. 
+
+## License
+Pallets are licensed under the [BUSL-1.1](https://spdx.org/licenses/BUSL-1.1.html) license, while the rest of the Clamor source code is licensed under [The Unlicense](./LICENSE) license. 
```
I would probably go full BUSL

satendrapalsingh replied:
updated

satendrapalsingh committed:
Fix more review comments

remove Unlicense, just keep BUSL-1.1
